src/github/dto/update.practices.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UpdatePracticesDto {
+  @ApiProperty({
+    description: 'Practices can be strings or a single paragraph.',
+    example: ['practice1', 'practice2'],
+  })
+  @IsNotEmpty()
+  practices: string[];
```

Comment:
PullProofAI

The `UpdatePracticesDto` class looks well-defined with validation and Swagger annotations. It's good practice to include these validations for ensuring data integrity and documenting API endpoints effectively.

```diff
+ import { IsNotEmpty } from 'class-validator';
+ import { ApiProperty } from '@nestjs/swagger';

export class UpdatePracticesDto {
    @ApiProperty({
        description: 'Practices can be strings or a single paragraph.',
        example: ['practice1', 'practice2'],
    })
    @IsNotEmpty()
    practices: string[];
}
```



<!-- This is an auto-generated comment by PullProofAI -->